### PR TITLE
Git homework

### PR DESCRIPTION
**Pull request by ArtremBotlung**

### CODE REVIEW

File: lantern/git/homework.py [hunk 1]
```diff
@@ -15,23 +15,23 @@ def is_two_object_has_same_value(first: Any, second: Any) -> bool:
...
+    return type(first) == type(second)
 
 
 def is_two_objects_is_the_same_objects(first: Any, second: Any) -> bool:
     """
     If @first and @second has same type should return True
     In another case should return False
     """
-    pass
+    return first is second
```
Comment (imelnyk1347) [inline lantern/git/homework.py]: in this situation better: return id(first) == id (second)

File: lantern/git/homework.py [hunk 2]
```diff
@@ -97,15 +107,20 @@ def is_word_in_text(word: str, text: str) -> bool:
...
     Use loop to create list that contain int values from 0 to 12 except 6 and 7
     """
-    pass
 
+    a = []
+    for i in range(13):
+        a.append(i)
+    a.remove(6)
+    a.remove(7)
+    return a
```
Comment (imelnyk1347) [inline lantern/git/homework.py:123]: to many programe code, you can do this shorter:
exepshen = [6, 7]
return[i for i in range(13) if i is not in exepshen]

File: lantern/git/homework.py [hunk 3]
```diff
@@ -116,7 +131,13 @@ def remove_from_list_all_negative_numbers(data: List[int]) -> list:
         remove_from_list_all_negative_numbers([1, 5, -7, 8, -1])
         >>> [1, 5, 8]
     """
-    pass
+    filter_data = []
+    for i in data:
+        if i>0:
+            filter_data.append(i)
+    return filter_data
```
Comment (imelnyk1347) [inline lantern/git/homework.py:138]: to many programe code, you can do this shorter.

File: lantern/git/homework.py [hunk 4]
```diff
@@ -127,7 +148,10 @@ def alphabet() -> dict:
         alphabet()
         >>> {"a": 1, "b": 2 ...}
     """
-    pass
+    a = {}
+    for i in range(26):
+        a[i + 1] = chr(ord('a') + i)
+    return a
```
Comment (imelnyk1347) [inline lantern/git/homework.py:154]: in this task you can import string library

**Review by imelnyk1347 — COMMENTED** [review]
try fix tasks

**Review by Zaplatynskyi — APPROVED** [review]
looks good to me

**Pull request merged**